packages/preview/clean-uoft-thesis/0.1.0/content/main.typ

```diff
@@ -0,0 +1,13 @@
+#import "@preview/clean-uoft-thesis:0.1.0": *
+
+#show: uoft.with(
+  title: "Title of Thesis",
+  author: "Firstname Lastname",
+  department: "Physiology",
+  degree: "Doctor of Philosophy",
+  graduation_year: "2025",
+  abstract: include("abstract.typ"),
```

Comment:
[`include` is a keyword](https://typst.app/docs/reference/scripting/#modules), not a function, so the parenthesis are not necessary here.